lib/services/causes_service.dart

```diff
@@ -93,11 +83,9 @@ class CausesService {
   /// Input Action id
   /// Returns the Action with that id
   Future<Campaign> getCampaign(int id) async {
-    final (response, userInfo) = await (
-      _causeServiceClient.getCampaignsApi().campaignsRetrieve(id: id),
-      getUserInfo(),
-    ).wait;
-    return Campaign.fromApiModel(response.data!, userInfo);
+    final response =
+        await _causeServiceClient.getCampaignsApi().campaignsRetrieve(id: id);
```

Comment:
I think it's more of a preference, but for me this looks more readable than this line break:
```suggestion
    final response = await _causeServiceClient
        .getCampaignsApi()
        .campaignsRetrieve(id: id);
```

Reply:
I agree but dartfmt doesn't seem to 😂. If I change this to what you suggested then dartfmt just changes it back...

Do you think there is a way we can configure this. From a very brief search it seems dart fmt does what it wants and you have very little say.